src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionBase.cs

```diff
@@ -24,5 +24,30 @@ public bool LifetimeExpired(long nowTicks, TimeSpan lifetime)
             if (expired && NetEventSource.IsEnabled) Trace($"Connection no longer usable. Alive {TimeSpan.FromMilliseconds((nowTicks - CreationTickCount))} > {lifetime}.");
             return expired;
         }
+
+        /// <summary>Awaits a task, ignoring any resulting exceptions.</summary>
+        internal static async Task IgnoreExceptionsAsync(ValueTask<int> task)
+        {
+            try { await task.ConfigureAwait(false); } catch { }
+        }
+
+        /// <summary>Awaits a task, ignoring any resulting exceptions.</summary>
+        internal static async Task IgnoreExceptionsAsync(Task task)
+        {
+            try { await task.ConfigureAwait(false); } catch { }
+        }
+
+        /// <summary>Awaits a task, logging any resulting exceptions (which are otherwise ignored).</summary>
+        internal async Task LogExceptionsAsync(Task task)
```

Comment:
I suggest making this be:
```C#
internal void LogExceptions(Task task)
{
    _ = LogExceptionsAsync(task);

    async Task LogExceptionsAsync(task)
    {
        ... // current implementation
    }
}
```
That way, call sites can just do `LogExceptions(...);` rather than `_ = LogExceptionsAsync(task);`.

Reply:
Can we do the same for IgnoreExceptionsAsync as well?